tests/fixtures/Helpers.cs

```diff
@@ -1,11 +1,21 @@
 using System;
-using TagLib;
+using System.IO;
 
 namespace TagLib.Tests
 {
 	public class Debugger
```

Comment:
I'm not sure I'd call this `Debugger` or at least not with test constants in here.